docs/core/concepts/mapping.md

```diff
@@ -1,6 +1,6 @@
 # Mapping
 
-Prefect introduces a flexible map/reduce model for dynamically executing parallel tasks.
+Prefect introduces a flexible map/reduce model for dynamically executing parallel tasks (parallel in the sense of independent data handling for each task, not parallelized simultaneous execution).
```

Comment:
This is a good catch.

Perhaps take this further and, instead of qualifying parallel (because execution could, in fact, be parallelized _or_ distributed), let's take parallel out of this introductory sentence entirely.

I'm thinking rewriting as something like: "a flexible map/reduce model for dynamically executing tasks across an iterable input. This, in turn, gives you the ability to execute mapped tasks in a distributed or parallel manner using an [executor](link) like the [DaskExecutor](link)."

I don't have a problem with linking users to the more advanced material, but you're right that we should explain clearly first.

Reply:
I like your rewrite @tpdorsey. The main reason I wanted to qualify 'parallel' instead of removing it, was because it appears multiple times later on in this page. Do you feel it is not a problem in that sense?

Reply:
I think that can work, I'll work that in and commit in a bit.